src/shared/sam/debugger/typescriptSamDebug.ts

```diff
@@ -118,6 +118,10 @@ export async function makeTypescriptConfig(config: SamLaunchRequestArgs): Promis
  */
 async function compileTypeScript(config: NodejsDebugConfiguration): Promise<void> {
     if (config.invokeTarget.target === 'code') {
+        const tscResponse = await new ChildProcess(true, 'tsc -v').run()
+        if (!tscResponse.stdout.startsWith('Version') ) {
+            throw new Error('The TypeScript compiler was not found. To install globally on your machine, run "npm install -g typescript" in the terminal.')
```

Comment:
If we are going to give instructions then we are in localization territory. If we are going to surface this as Error() let's keep it precise yet concise.
```suggestion
            throw new Error('TypeScript compiler "tsc" not found.')
```